plugins/importexport/medra/classes/form/MedraSettingsForm.inc.php

```diff
@@ -41,17 +84,29 @@ function MedraSettingsForm(&$plugin, $journalId) {
 		$this->addCheck(new FormValidatorInSet($this, 'publicationCountry', FORM_VALIDATOR_REQUIRED_VALUE, 'plugins.importexport.medra.settings.form.publicationCountry', array_keys($this->_getCountries())));
 		// The username is used in HTTP basic authentication and according to RFC2617 it therefore may not contain a colon.
 		$this->addCheck(new FormValidatorRegExp($this, 'username', FORM_VALIDATOR_OPTIONAL_VALUE, 'plugins.importexport.medra.settings.form.usernameRequired', '/^[^:]+$/'));
+		$this->addCheck(new FormValidatorPost($this));
```

Comment:
Once the CSRF stuff is merged in, we'll need to add the CSRF checks to the code in this PR too -- let's talk about that later.